roles/openshift_aws/templates/launchinstances.json.j2

```diff
@@ -0,0 +1,39 @@
+{
+    "Version": "2012-10-17",
+    "Statement": [
+	{
+	    "Action": [
+		"ec2:DescribeAvailabilityZones",
+		"ec2:DescribeImages",
+		"ec2:DescribeInstances",
+		"ec2:DescribeKeyPairs",
+		"ec2:DescribeSecurityGroups",
+		"ec2:RunInstances",
+		"ec2:StartInstances",
+		"ec2:DescribeInstances",
+		"ec2:CreateVolume",
+		"ec2:AttachVolume",
+		"ec2:DetachVolume"
```

Comment:
additional permissions required:
```
ec2:DescribeVpcs
ec2:DescribeSubnets
```
We also need the iam:PassRole permission to set the IAM role on the created instance
(see https://aws.amazon.com/blogs/security/granting-permission-to-launch-ec2-instances-with-iam-roles-passrole-permission/)

I'm not clear on how we'd specify the role if we don't necessarily have a urn for it